app/viewModels/drafts/AllDraftDeparturesViewModel.scala

```diff
@@ -69,6 +77,40 @@ case class AllDraftDeparturesViewModel(items: DeparturesSummary,
 
   def pageNumber: Int = paginationViewModel.pageNumber
 
+  def sortLrn: String = sortParams match {
+    case SortByLRNAsc  => "ascending"
+    case SortByLRNDesc => "descending"
+    case _             => "none"
+  }
+
+  def sortCreatedAt: String = sortParams match {
+    case SortByCreatedAtAsc  => "ascending"
+    case SortByCreatedAtDesc => "descending"
+    case _                   => "none"
+  }
```

Comment:
This would tie in well with the backend refactor I proposed, as both of these would be:
```
sortParams match {
  case Ascending => "ascending"
  case Descending => "descending"
}
```
But this could be refactored further. Perhaps overriding `toString` in the `Order` traits and just calling `sortParams.toString`